articles/commerce/pricing-apis.md

```diff
@@ -527,4 +550,190 @@ The following table shows the input parameters for the *RemoveCoupons* API.
+                    "Description": "",
+                    "ValidFromDate": "1900-01-01T00:00:00.0000000Z",
+                    "ValidToDate": "2154-12-31T00:00:00.0000000Z",
+                    "CouponCodes": [],
+                    "DateValidationTypeValue": 1
+                }
+            ]   
+        },
+        {
+            "ProductId": 68719489872,
+            "ProductDiscounts":
+            [
+                {
+                    "OfferId": "ST100009",
+                    "OfferName": "Student discount",
+                    "PeriodicDiscountTypeValue": 2,
+                    "IsDiscountCodeRequired": false,
+                    "ValidationPeriodId": null,
+                    "AdditionalRestrictions": null,
+                    "Description": "Students get 10% off on Jeans and Backpacks",
+                    "ValidFromDate": "1900-01-01T00:00:00.0000000Z",
+                    "ValidToDate": "2154-12-31T00:00:00.0000000Z",
+                    "CouponCodes": [],
+                    "DateValidationTypeValue": 1
+                }
+            ]   
+        }...
+    ]
+}
+```
+</details>
+
+See [PriceLookupContext section](#PriceLookupContext) for details.
+
+### PriceLookupContext
+
+A class used for property-base pricing model.
+
+It is now used in *GetProductPromotions* and *GetActivePrices*
+
+Structure of PriceLookupContext
+```json
+{
+    HeaderContext: PriceLookupHeaderContext
+    {
+        CustomerAccountNumber: string
+        AffiliationLoyaltyTierLines: IEnumerable<AffiliationLoyaltyTier>
+        ChannelId: long?
+        SalesOrderProperties: IEnumerable<AttributeValueBase>
+    },
+    LineContexts: IEnumerable<PriceLookupLineContext>
+    [
+        {
+            ProductRecordId: string
+            UnitOfMeasureSymbol: string
+            InventorySiteId: string
+            InventoryLocationId: string
+            DeliveryMode: string
+            CatalogId: string
+            SalesLineProperties: IEnumerable<AttributeValueBase>
+        },
+    ]
+}
+```
+
+Sample
+```json
+"PriceLookupContext":
+{
+    "HeaderContext": 
+    {
+        "CustomerAccount": 2001,
+        "AffiliationLoyaltyTierLines": 
+        [
+            {
+                "AffiliationId": 5637144577,
+                "LoyaltyTierId": 0, 
+                "AffiliationTypeValue": 0,
+                "ReasonCodeLines": [],
+                "CustomerId": "2001"
+            }
+        ],
+        "SalesOrderProperties":
+        [
+            {
+                "@odata.type": "#Microsoft.Dynamics.Commerce.Runtime.DataModel.AttributeTextValue",
+                "Name": "CalcDate",
+                "TextValue": "2022-10-10"
+            }
+        ]
+    },
+    "LineContexts": []
+}
+```
+
+Remark:
+In PriceLookupHeaderContext, we do not have customer group as it would be indicate by customer number.
+In PriceLookupHeaderContext, you can indicate channelId otherwise channelId from request context would be used.
```

Comment:
ChannelId could be specified. If it is not specified, the one from request context would be used.

Also add explanation as to what is the channelId in request context.